source/Symbol/SwiftASTContext.cpp

```diff
@@ -2992,20 +2993,44 @@ swift::ASTContext *SwiftASTContext::GetASTContext() {
     std::string ModuleCachePath = GetClangImporterOptions().ModuleCachePath;
     StringRef PrebuiltModuleCachePath =
       GetCompilerInvocation().getFrontendOptions().PrebuiltModuleCachePath;
-    std::unique_ptr<swift::ModuleLoader> parseable_module_loader_ap(
-      swift::ParseableInterfaceModuleLoader::create(
-        *m_ast_context_ap, ModuleCachePath, PrebuiltModuleCachePath,
-        /*tracker=*/nullptr, swift::ModuleLoadingMode::PreferSerialized));
 
-    if (parseable_module_loader_ap) {
-      m_parseable_module_loader =
-      (swift::ParseableInterfaceModuleLoader *)parseable_module_loader_ap.get();
-      m_ast_context_ap->addModuleLoader(std::move(parseable_module_loader_ap));
+    auto MLM = swift::ModuleLoadingMode::PreferSerialized;
+    if (auto forceModuleLoadingMode =
```

Comment:
Nitpick: this should use the LLDB variable naming style, even though it's mostly copied from Swift. (Alternately, we could wrap this up in a helper function that LLDB can call.)